src/email/email.service.ts

```diff
@@ -0,0 +1,76 @@
+import nodemailer from 'nodemailer';
+import env from '../config';
+
+const { SMPT_USERNAME, SMPT_PASSWORD, FRONT_URL } = env;
+
+export const EmailService = new (class {
+    async registrationEmail(to: string, username: string, userId: string) {
+        await this.sendEmail(
+            to,
+            'Registration email in TeamUp',
```

Comment:
these templates are kinda messy, extract these to separate utility functions imo

Like
`const registrationTemplate = (userName: string, userId: string) => '<template>'`

Much more cleaner imo

Reply:
maybe we should create a separate directory `src/email/templates/` with `*.template.html` files. It will be cleaner and will allow for syntax highlighting in html files. It is bad to keep constant strings like that in places like this.